Avalonia.MusicStore/ViewModels/MainWindowViewModel.cs

```diff
@@ -1,40 +1,40 @@
 ﻿using Avalonia.MusicStore.Models;
-using ReactiveUI;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using System;
 using System.Collections.ObjectModel;
 using System.Linq;
-using System.Reactive.Linq;
-using System.Windows.Input;
-using System.Reactive.Concurrency;
+using System.Threading.Tasks;
 
 namespace Avalonia.MusicStore.ViewModels
 {
-    public class MainWindowViewModel : ViewModelBase
+    public partial class MainWindowViewModel : ObservableObject
     {
-        public Interaction<MusicStoreViewModel, AlbumViewModel?> ShowDialog { get; }
+        public ObservableCollection<AlbumViewModel> Albums { get; } = new();
 
-        public ICommand BuyMusicCommand { get; }
+        public Func<MusicStoreViewModel, Task<AlbumViewModel?>>? OnShowDialog { get; set; }
 
         public MainWindowViewModel()
         {
-            RxApp.MainThreadScheduler.Schedule(LoadAlbums);
+            Task.Run(LoadAlbums);
```

Comment:
Nit: We should avoid Task.Run in the constructor. Old code wasn't ideal either. If possible, we should load albums from the Window.OnLoaded method, LostAlbums can be a command in this case. 

But that's debatable.